edkrepo_manifest_parser/edk_manifest.py

```diff
@@ -378,7 +394,34 @@ def get_combo_element(self, name):
         for combo in combinations.iter(tag='Combination'):
             if combo.attrib['name'] == name:
                 return copy.deepcopy(combo)
-        raise ValueError(COMB_UNKOWN_ERROR.format(name, self._fileref))
+        raise ValueError(COMB_UNKNOWN_ERROR.format(name, self._fileref))
+
+    def get_all_patchsets(self):
+        patchsets = []
+        for patch in self._patch_sets.keys():
+            patchsets.append(self._patch_sets[patch])
+        return patchsets
+
+    def get_parent_patchset_operations(self, name, patch_set_operations):

```

Comment:
please add a docstring to this method describing the input parameters and the format and content of the return value